.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main,remote-db-ls-cache]
```

```suggestion
    branches: [main]
```

apply the suggestion to remove current branch before merging